src/test/java/core/basesyntax/operation/BalaceOperationHandlerTest.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.operation;
+
+import static org.junit.Assert.assertEquals;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.storage.Storage;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalaceOperationHandlerTest {
+    private BalaceOperationHandler balaceOperationHandler;
+
+    @Before
+    public void setUp() {
+        balaceOperationHandler = new BalaceOperationHandler();
+    }
+
+    @Test
+    public void apply_validFruitTransaction_Ok() {
+        FruitTransaction fruitTransaction = new FruitTransaction("b", new Fruit("banana"), 10);
```

Comment:
Make the hardcoded values such as "b", "banana" (or even the whole Fruit object), int 10 constants